stock_picking_delivery_rate/README.rst

```diff
@@ -39,7 +39,7 @@ Bug Tracker
 ===========
 
 Bugs are tracked on `GitHub Issues
-<https://github.com/OCA/stock-logistics-workflow/issues>`_. In case of trouble, please
+`<https://github.com/OCA/carrier-delivery/issues>`_. In case of trouble, please
```

Comment:
delivery-carrier

Also, looks like this is wrong on  OCA/9.0. Can you please submit a quick hotfix for that?